frontend/ui/widget/keyvaluepage.lua

```diff
@@ -467,20 +381,112 @@ function KeyValuePage:init()
     }
     self.item_width = self.inner_dimen.w
 
-    local footer = BottomContainer:new{
-        dimen = self.inner_dimen:copy(),
-        self.page_info,
-    }
-    local page_return = BottomContainer:new{
-        dimen = self.inner_dimen:copy(),
-        WidgetContainer:new{
-            dimen = Geom:new{
-                w = self.inner_dimen.w,
-                h = self.return_button:getSize().h,
+    local footer -- will stay nil if self.single_page is true
+    if self.single_page then
+        -- group for page info
+        local chevron_left = "chevron.left"
```

Comment:
I guess you wanted to write `if NOT self.single_page` :)

(That's a lot of changes/diff because (only?) of some indentation changes. I always have some moral dilemna when that happens to me :) I would  probably have gone in this case with just creating all these whether single_page or not, and just not use it if single_page (with may be a footer:free() to be sure) - as single_page is a super rare use case for now.)